src/Core/src/Platform/Windows/FrameworkElementExtensions.cs

```diff
@@ -0,0 +1,144 @@
+﻿#nullable enable
+using System;
+using System.Linq;
+using Microsoft.UI.Xaml;
+using System.Collections.Concurrent;
+using Microsoft.UI.Xaml.Media;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.UI.Xaml.Controls;
+using WBinding = Microsoft.UI.Xaml.Data.Binding;
+using WBrush = Microsoft.UI.Xaml.Media.Brush;
+using WBindingExpression = Microsoft.UI.Xaml.Data.BindingExpression;
+
+namespace Microsoft.Maui
+{
+	public static class FrameworkElementExtensions
```

Comment:
Do we want this to be public ?

Reply:
Was internal in Xamarin.Forms, should we changed it?, any reason to not be public?